src/main/java/uk/gov/companieshouse/company/metrics/config/ApplicationConfig.java

```diff
@@ -26,13 +28,11 @@ EnvironmentReader environmentReader() {
     }
 
     @Bean
-    Supplier<InternalApiClient> internalApiClientSupplier(
-            @Value("${api.company-metrics-api-key}") String apiKey,
-            @Value("${api.api-url}") String apiUrl) {
-        return () -> {
+    BiFunction<String, String, InternalApiClient> internalApiClientSupplier() {
```

Comment:
We dont need a bifuntion as api url and key will always be same for metrics-api and charges-data-api, we can revert to old code and remove new config all together maybe?

Reply:
reverted back to old code with Supplier